### ISSUE TITLE
Skip to content

### ISSUE THREAD

**Issue by Blackbaud-BrandonJones** [blackbaud/stache2#483]
### Expected behavior
Accessibility when using a keyboard or screen reader, I should be able to skip reading / tabbing through all the navigation before getting to the page content.
### Actual behavior
pages require one to tab through or read through all the nav items before being able to access page content.

Feature request: Add a `Skip to content` link at the top of our layouts.
